action.yml

```diff
@@ -1,14 +1,13 @@
 name: "Python Poetry Action"
-author: "@abatilo"
+author: "@abatilo, @marius-cristian"
 description: "An action to run https://github.com/python-poetry/poetry"
 branding:
   icon: "truck"
   color: "gray-dark"
 inputs:
   poetry-version:
     description: "The version of poetry to install"
-    required: false
-    default: "latest"
```

Comment:
No default.
We always force pin a version.
It will add a bit of developer friction, but more control of the supply chain